lib/OpenQA/Script/CloneJob.pm

```diff
@@ -241,10 +241,38 @@ sub clone_jobs ($jobid, $options) {
     handle_tx($tx, $url_handler, $options, $jobs) if $tx;
 }
 
+sub detect_maintenance_update ($job) {
+    my $test_settings = $job->{settings};
+    my @maintenance_list;
+    my $ua = LWP::UserAgent->new;
+    $ua->timeout(10);
+    for my $test_setting (%$test_settings) {
+        if ($test_setting =~ /(.*)TEST_TEMPLATE/) {
+            next unless $test_settings->{$1 . 'TEST_ISSUES'};
+            foreach my $test_issue (split(/,/, $test_settings->{$1 . 'TEST_ISSUES'})) {
+                my $test_template_url = $test_settings->{$test_setting};
+                $test_template_url =~ s/\@INCIDENTNR\@/$test_issue/;
+                my $response = $ua->get($test_template_url);
+                if (!$response->is_success) {
+                    push @maintenance_list, $test_template_url;
+                }
+            }
+        }
+    }
+
+    if (@maintenance_list) {
+        print "Current job will fail, because the below updates have already been released\n", pp(@maintenance_list);
+    }
```

Comment:
@DeepthiYV make it die at the end, there's no need to continue if we alrready know it's gonna fail :), update the ticket accordinglly